groups/chd-infrastructure/locals.tf

```diff
@@ -0,0 +1,46 @@
+# ------------------------------------------------------------------------
+# Locals
+# ------------------------------------------------------------------------
+locals {
+  admin_cidrs = values(data.vault_generic_secret.internal_cidrs.data)
+
+  internal_fqdn = format("%s.%s.aws.internal", split("-", var.aws_account)[1], split("-", var.aws_account)[0])
+
+  default_tags = {
+    Terraform = "true"
+    Region    = var.aws_region
+    Account   = var.aws_account
+  }
+
+# ------------------------------------------------------------------------------
+# CHD BEP
+# ------------------------------------------------------------------------------
+#  chd_bep_data = data.vault_generic_secret.chd_bep_data.data_json
```

Comment:
Also worth cleaning this up if no longer required.

Reply:
This will be required in the future but is not populated right now.